### ISSUE TITLE
Identify duplicate computers using their SMBIOS GUID

### ISSUE THREAD

**Issue by pLeXwithFleX** [chocolatey/chocolatey-licensed-issues#367]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Is Your Feature Request Related To A Problem? Please describe.

When a computer is reinstalling, the central management generates a duplicated computer object. This has a negative impact for used licenses and you have always to cleanup the computer objects.

### Describe The Solution. Why is it needed?

The chocolatey agent could determine the SMBIOS GUID from the windows registry and the central management could use this to identify a computer object. It should link the fresh installed system to the existing computer object and replace the installed software list.

┆Issue is synchronized with this [Gitlab issue](https://gitlab.com/chocolatey/central-management/choco-licensed-management-ui/-/issues/941) by [Unito](https://www.unito.io)

**Comment by pauby** [comment]
So we can capture all the information for this, can I ask you to raise a new issue in this repository, using the correct template (which will ask you for all the information needed) and referencing this issue? Once that is done, we can close this issue (as it will no longer be needed).

**Comment by pLeXwithFleX** [comment]
Hi pauby,

youre welcome, Im sorry for posting to the wrong repository. I will create a new issue.
